uitests/tests/org.jboss.tools.fuse.ui.bot.tests/pom.xml

```diff
@@ -30,7 +30,7 @@
 		<staging.repos>false</staging.repos>
 		<rd.skipUnfixedIssues>true</rd.skipUnfixedIssues>
 		<skipUITests>true</skipUITests>
-		<skipRequirements>true</skipRequirements>
+		<skipRequirements>false</skipRequirements>
```

Comment:
out of curiosity, why this value has been changed?

Reply:
This is for downloading a karaf instance which can be used in tests. Since the tests are public, they should be executable with some public requirements such as the karaf instance.

Reply:
ok thanks